content/3_Silver/2P.mdx

```diff
@@ -31,6 +31,144 @@ frequency: 2
 	</Resource>
 </Resources>
 
+## Two Pointers
+
+The two pointers method iterates two pointers across an array, to track the start and end
+of an interval, or two values in a sorted array that we are currently checking. 
+Typically, each pointer starts at seperate ends and moves strictly in one direction. 
+However, as shown in the focus problem below, the two pointers do not have to start at seperate ends.
```

Comment:
oops I found this rather confusing (no idea how to fix though) D:

Reply:
which part is confusing?